packages/react-components/src/components/AppFrame/components/SideNavigationItem/types.ts

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+
+import { ComponentCoreProps } from '../../../../utils/types';
+
+export interface ISideNavigationItemProps extends ComponentCoreProps {
+  /**
+   * The label of the item
+   */
+  label: React.ReactNode;
+  /**
+   * It will display the custom element on the right side of the side navigation item
```

Comment:
"It will display the custom element..." is janky wording, IMO. Could we go for "Displays the custom element...", or, better yet, "Custom element"? Same for other cases where similar wording appears.